src/lib/types/openapiDoc.ts

```diff
@@ -6525,7 +6525,7 @@ const document = {
                         description: 'The endpoint HTTP method.',
                       },
                       path: { type: 'string', description: 'The endpoint path.' },
-                      schema: { nullable: true },
+                      schema: { type: 'string', nullable: true },
```

Comment:
The lack of type here was breaking frontmatter validation with the error "nullable cannot be used without type" from ajv

Reply:
😬 I actually had to fix something similar for this over as part of my backgrounding work in https://github.com/readmeio/readme/pull/14881/commits/b09cb92c36831a1ca5ccc18fa035f8f663fc5319.

Since `type: string` is incorrect for this data shape, as it's a nullable object. Does `type: object` pass AJV here instead? Or if that doesn't work what about `additionalProperties: {}`?

```suggestion
                      schema: { type: 'object', properties: {}, nullable: true },
```

```suggestion
                      schema: { additionalProperties: {}, nullable: true },
```

Really wish I had just returned the schema as a string...

Reply:
`schema: { type: 'object', properties: {}, nullable: true },` works!

Reply:
awesome, i'll see if i can update our OAS generation to make that change there so we don't have to worry about preserving it just here.

Reply:
ope, just seeing this. yeah if we could defer to the auto-generated OAS as our source of truth, i'd prefer that approach. ([more info in this comment](https://github.com/readmeio/rdme/pull/1261#discussion_r2151158098))